cluster/gce/util.sh

```diff
@@ -836,6 +851,12 @@ function delete-network() {
 
 function delete-subnetworks() {
   if [[ ${ENABLE_IP_ALIASES:-} != "true" ]]; then
+    echo "Deleting default subnets..."
+    # This value should be kept in sync with number of regions.
+    local parallelism=9
+    gcloud compute networks subnets list | \
+    grep "${NETWORK}" | tr -s " " | cut -f2 -d" " | \
```

Comment:
Yes it will because it has to (at least that's what I figured during testing). When you change network type to 'custom' you need to delete all subnets, including auto ones, to be able to delete network itself.